views/views.go

```diff
@@ -0,0 +1,17 @@
+//go:generate go run github.com/a-h/templ/cmd/templ@v0.2.543 generate
+package views
+
+import (
+	"encoding/json"
+)
+
+func ToJSON(pairs ...string) string {
```

Comment:
@verheyenkoen So, this is only used for setting the snapshot ID as an `If-Match` header? e.g. ` hx-headers={ views.ToJSON("If-Match", dataset.SnapshotID) }`.  I don't think having a helper function that creates a simple key-value pair adds value here, unless we can argue that there's reuse elsewhere.

That's why the original code simply explicitly creates a JSON string: `hx-headers='{"If-Match": "{{.Dataset.SnapshotID}}"}'`. It's good enough to accomplish what it needs to do.

@nics

Reply:
Resolved using:
```go
hx-headers={ fmt.Sprintf(`{"If-Match": "%s"}`, dataset.SnapshotID) }
```